handbook/engineering/code-intelligence/goals.md

```diff
@@ -25,46 +27,110 @@ Progress on adoption and usage is tracked in our [Looker dashboard](https://sour
     
     **Outcome:** 5 popular open source C++ repositories have up-to-date precise code intelligence on Sourcegraph Cloud.
 
-1. 🔄 Sourcegraph Cloud has precise code intelligence for C++ repositories that use cmake as a build system. 
+1. ✅ Sourcegraph Cloud has precise code intelligence for C++ repositories that use cmake as a build system. 
     - ✅ Auto-index 5 repositories on Sourcegraph Cloud that use cmake as a build system.
     - ✅ Research: Add inference rules for cmake projects to auto-index on Cloud.
-    - 🔄 Auto-detect C++/cmake repositories and display common configurations and documentation to guide users in their set-up.
-    
+   
      **Outcome:** 5 popular open source cmake repositories have up-to-date precise code intelligence on Sourcegraph Cloud.
 
+1. Build lsif-clang testing infrastructure
+    - TBD roadmap item
```

Comment:
If these are TBD does it make sense to move to after the Bazel milestone so we can set the steps out?

Reply:
@tjdevries and @gbrik were just syncing on this when I opened the PR. I'll update with the new steps, but we can change the order anyway if we feel that's the best way forward.